imsv-docs-astro/astro.config.mjs

```diff
@@ -36,16 +36,20 @@ export default defineConfig({
           label: 'Guides',
           items: [
             { label: 'Introduction', link: 'guides' } ,
+            { label: 'Authentication', link: 'guides/authentication' } ,
+            { label: 'HTTP Status Codes', link: 'guides/http-status-codes' } ,
+            { label: 'Webhooks', link: 'guides/webhooks' } ,
+            { label: 'Cardholder Support', link: 'guides/cardholder-support' } ,
             { label: 'Core Concepts', autogenerate: { directory: 'guides/core-concepts' } },
             { label: 'Card Issuing Apps', autogenerate: { directory: 'guides/card-issuing-apps' } },
-            { label: 'KYC Modes' , autogenerate : { directory: 'guides/kyc-modes' } },
+            { label: 'KYC Modes' , items: [
```

Comment:
had to manually put items, because couldn't figure out sorting - intro page was in the bottom